modules/app.xql

```diff
@@ -468,15 +472,15 @@ declare function app:details-language($node as node(), $model as map(*)) as node
 :)
 declare function app:browse-newspaper($node as node(), $model as map(*)) as node()+ {
       let $collection := collection:collection()
```

Comment:
I don't believe `$collection` is needed anymore, since it's been replaced by the `collection:document` call on line 483.

Reply:
Good catch!